app/Services/Contact/Reminder/CreateReminder.php

```diff
@@ -43,39 +41,23 @@ public function execute(array $data) : Reminder
     {
         $this->validate($data);
 
-        Contact::where('account_id', $data['account_id'])
+        $contact = Contact::where('account_id', $data['account_id'])
             ->findOrFail($data['contact_id']);
 
-        if ($data['special_date_id']) {
-            SpecialDate::where('account_id', $data['account_id'])
-                ->findOrFail($data['special_date_id']);
-        }
-
-        $reminder = $this->attachReminderToLifeEvent($data);
-
-        $reminder->calculateNextExpectedDate()->save();
-        $reminder->scheduleNotifications();
-
-        return $reminder;
-    }
+        $reminder = Reminder::create([
+            'account_id' => $data['account_id'],
+            'contact_id' => $data['contact_id'],
+            'title' => $data['title'],
+            'description' => $this->nullOrValue($data, 'description'),
+            'initial_date' => $data['initial_date'],
+            'frequency_type' => $data['frequency_type'],
+            'frequency_number' => $data['frequency_number'],
+            'delible' => (isset($data['delible']) ? $data['delible'] : true),
```

Comment:
Thanks to Sonar:
```suggestion
            'delible' => (! isset($data['delible']) || $data['delible']),
```

Reply:
Ok I agree BUT the fix makes it way harder to understand. My version is actually easier to understand. With your fix, you have to think: ok, it's a negative of something that might be missing OR if the data is actually there... wait...